network.go

```diff
@@ -88,18 +88,38 @@ func updateInterfaceMTU(netHandle *netlink.Handle, link netlink.Link, mtu int) e
 }
 
 func updateLink(netHandle *netlink.Handle, link netlink.Link, iface *pb.Interface, actionType pb.UpdateType) error {
-	switch actionType {
-	case pb.UpdateType_AddIP:
+	knownActionType := false
+
+	// Add IPs
+	if (actionType & pb.UpdateType_AddIP) == pb.UpdateType_AddIP {
+		knownActionType = true
 		return updateInterfaceAddrs(netHandle, link, iface.IpAddresses, true)
```

Comment:
`return` will stop the process, to achieve this purpose, I think you should call

```
if err := updateInterfaceAddrs(netHandle, link, iface.IpAddresses, true); err != nil {
    // log or return error?
}
```